views/js/controller/runtime/testRunner.js

```diff
@@ -372,7 +372,10 @@ define([
              */
             killItemSession : function (callback) {
                 testMetaData.addData({
-                    'ITEM' : {'ITEM_END_TIME_CLIENT' : Date.now() / 1000}
+                    'ITEM' : {
+                        'ITEM_END_TIME_CLIENT' : Date.now() / 1000,
+                        'ITEM_TIMEZONE' : moment().utcOffset(moment().utcOffset()).format('Z') + ' ' + /\((.*)\)/.exec(new Date().toString())[1]
+                    }
```

Comment:
Do we really need the timezone abbreviation ? The form  `+02:00` or `+02` is ISO-8601